src/libraries/System.Private.Xml/src/System/Xml/Serialization/Types.cs

```diff
@@ -87,7 +87,6 @@ internal sealed class TypeDesc
         private TypeDesc? _nullableTypeDesc;
         private readonly TypeKind _kind;
         private readonly XmlSchemaType? _dataType;
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
Why remove these annotations? Typically we try to keep them if they "work" so if new code starts calling the methods it is annotated correctly.

Reply:
Would the newly added DAMT annotations be better for this (once I get that merged)? (I don't have an opinion about keeping the annotations or removing or updating it since I don't know what any of this is for - just bringing up one more option).